### DISCUSSION TITLE
How to add a selected value after addOption?

### DISCUSSION THREAD

**Discussion by JulcarlSelma** [sa-si-dev/virtual-select#173] [Q&A] [answered]
Hi!, How can I do something like this? I want to select new added option but I can't do it like this. how to do it?
![image](https://user-images.githubusercontent.com/20558190/186808888-de50f521-60c2-45af-b48a-99e921fbacd0.png)

**Comment by sa-si-dev** [comment, marked as answer]
Use [setValue](https://sa-si-dev.github.io/virtual-select/#/methods?id=setvalue) method.
